java/parse-interface/src/main/java/org/sqlflow/parser/parse/ParseResult.java

```diff
@@ -11,4 +11,6 @@
   public int position;
   // Errors encountered during parsing.
   public String error;
+  // tables that each statement manipulates.
+  public ArrayList<InputOutputTables> inputOutputTables;
```

Comment:
Similarly, use `List`

Reply:
Original `public ArrayList<String> statements;` is still using `ArrayList` will modify this in another PR.